.buildkite/pipeline-utils/ci-stats/pick_test_group_run_order.ts

```diff
@@ -16,8 +16,11 @@ import { BuildkiteClient, BuildkiteStep } from '../buildkite';
 import { CiStatsClient, TestGroupRunOrderResponse } from './client';
 
 import DISABLED_JEST_CONFIGS from '../../disabled_jest_configs.json';
+import { serverless, stateful } from './ftr_configs_manifests.json';
```

Comment:
Using json to store manifests paths as importing constant from `.buildkite/pipeline-utils` into `packages/kbn-test` won't work and I don't like the idea of solving it with ` ts-ignore`. Open for suggestions here